horace_core/spaghetti_plot.m

```diff
@@ -248,14 +248,14 @@
             u2crt = cross(u1crt,b*[1;0;0]./norm(b*[1;0;0]));
         end
     else
-        u2crt = u2crt0./norm(u2crt0);
+        u2crt = ortho_vec(u1crt);
```

Comment:
wondering where ortho_vec is - it's not in this file or in the existing codebase

Reply:
its another ticket in Herbert